src/videouplinkbandwidthpolicy/DefaultSimulcastUplinkPolicy.ts

```diff
@@ -0,0 +1,379 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import Logger from '../logger/Logger';
+import Maybe from '../maybe/Maybe';
+import AsyncScheduler from '../scheduler/AsyncScheduler';
+import SimulcastLayers from '../simulcastlayers/SimulcastLayers';
+import SimulcastTransceiverController from '../transceivercontroller/SimulcastTransceiverController';
+import DefaultVideoAndEncodeParameter from '../videocaptureandencodeparameter/DefaultVideoCaptureAndEncodeParameter';
+import VideoStreamDescription from '../videostreamindex/VideoStreamDescription';
+import VideoStreamIndex from '../videostreamindex/VideoStreamIndex';
+import BitrateParameters from './BitrateParameters';
+import ConnectionMetrics from './ConnectionMetrics';
+import SimulcastUplinkObserver from './SimulcastUplinkObserver';
+import SimulcastUplinkPolicy from './SimulcastUplinkPolicy';
+
+const enum ActiveStreams {
+  kHiAndLow,
+  kMidAndLow,
+  kLow,
+}
+
+/**
+ * [[DefaultSimulcastUplinkPolicy]] determines capture and encode
+ *  parameters that reacts to estimated uplink bandwidth
+ */
+export default class DefaultSimulcastUplinkPolicy implements SimulcastUplinkPolicy {
+  static readonly defaultUplinkBandwidthKbps: number = 1200;
+  static readonly startupDurationMs: number = 6000;
+  static readonly holdDownDurationMs: number = 4000;
+  static readonly defaultMaxFrameRate = 15;
+
+  // Current rough estimates where webrtc disables streams
+  static readonly kHiDisabledRate = 700;
+  static readonly kMidDisabledRate = 240;
+
+  private numParticipants: number = 0;
+  private optimalParameters: DefaultVideoAndEncodeParameter;
+  private parametersInEffect: DefaultVideoAndEncodeParameter;
+
```

Comment:
Can we eliminate the blank lines between the variable declarations.
We can have a blank line between the last `static readonly` variable and the first `private` variable. Blank lines between the `private` variables is not required.

Reply:
Cool, removed the spaces, thanks!